src/modules/connections/connections.service.ts

```diff
@@ -66,4 +66,9 @@ export class ConnectionsService {
     return formatTablesAndColumnsResponse(result);
   }
 
+  async getConnectionsCount(admin: number) {
+    const count = await this.connectionRepository.getCount({ admin_id: admin });
+    return { totalConnections: count };
```

Comment:
Maybe use field count everywhere, instead of totalConnections, totalModels, etc?

Reply:
done